From a93643e6909906c12f6659e95b0bd10b7a7f925e Mon Sep 17 00:00:00 2001
From: geruijun <geruijun@huawei.com>
Date: Mon, 27 Dec 2021 19:43:22 +0800
Subject: [PATCH] PCI: Add support of port isolation for QLogic HBA card 26
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a93643e6909906c12f6659e95b0bd10b7a7f925e


euleros inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I4MD9D?from=project-issue
CVE: NA

--------------------------------

Add support of port isolation for QLogic HBA card 26

Signed-off-by: geruijun <geruijun@huawei.com>
Signed-off-by: Hongxiang Lou <louhongxiang@huawei.com>
Reviewed-by: Xiongfeng Wang <wangxiongfeng2@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/pci/quirks.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pci/quirks.c b/drivers/pci/quirks.c
index 7ec42a6ab964..54a8e390ad67 100644
--- a/drivers/pci/quirks.c
+++ b/drivers/pci/quirks.c
@@ -4890,6 +4890,8 @@ static const struct pci_dev_acs_enabled {
 	{ PCI_VENDOR_ID_INTEL, PCI_ANY_ID, pci_quirk_intel_spt_pch_acs },
 	{ 0x19a2, 0x710, pci_quirk_mf_endpoint_acs }, /* Emulex BE3-R */
 	{ 0x10df, 0x720, pci_quirk_mf_endpoint_acs }, /* Emulex Skyhawk-R */
+	{ 0x1077, 0x2532, pci_quirk_mf_endpoint_acs}, /* QLogic QL2562 */
+	{ 0x1077, 0x2261, pci_quirk_mf_endpoint_acs}, /* QLogic QL2692 */
 	{ 0x1077, 0x2031, pci_quirk_mf_endpoint_acs}, /* QLogic QL2672 */
 	{ 0x1077, 0x2532, pci_quirk_mf_endpoint_acs},
 	/* Cavium ThunderX */
-- 
2.26.2

